themes/digital.gov/layouts/communities/list.html

```diff
@@ -23,33 +23,98 @@ <h1>{{- .Title | markdownify -}} </h1>
+                    <h2 class="usa-card__heading">{{- $e.Title | markdownify -}}</h2>
+                  </div>
+                  <div class="usa-card__media">
+                    <div class="usa-card__img">
+                      <img
+                        src="{{- $e.Params.dg_image | absURL -}}"
+                        alt="{{- $e.Title }}"
+                      />
+                    </div>
+                  </div>
+                  <div class="usa-card__body">
+                    <p>
+                      {{- $e.Params.summary | markdownify -}}
+                    </p>
+                  </div>
+                  <div class="usa-card__footer">
+                    <a class="usa-button" href="{{- $e.Permalink -}}" title="{{- $e.Title -}}">
+                      Join&nbsp;{{- $e.Title -}}
+                    </a>
+                  </div>
+                </div>
+              </li>
+              {{ end }}
+          </div>
+      </div>
+    </div>
+  </section>
 
-        {{/* Gets all communities pages */}}
-        {{ $communities := (where .Site.Pages "Section" "communities").ByTitle }}
+  <div class="grid-container grid-container-desktop">
+    <div class="grid-row tablet-lg:grid-gap-4">
+      <div class="grid-col-12">
+        <header class="page-head page-head-{{- .Type -}}">
 
-        {{/* Gather only the pages with a weight greater than or equal to 1 */}}
-        {{ $communities := (where $communities ".Params.weight" "ge" 1) }}
+          {{/* Page Title */}}
+          <h1>Other Communities of Practice</h1>
 
-        {{ range $i, $e := $communities }}
-          <div class="grid-col-12 tablet:grid-col-6">
+        </header>
+      </div>
+    </div>
+  </div>
 
-            <article class="promo promo-card" data-edit-this="{{- .Path -}}">
-              <div class="grid-row tablet-lg:grid-gap-4">
-                <div class="grid-col-12">
-                  <h4><a href="{{- $e.Permalink -}}" title="{{- $e.Title | markdownify -}}">{{- $e.Title | markdownify -}}</a></h4>
-                  <p>{{- $e.Params.summary | markdownify -}}</p>
+  <section class="usa-section">
+    <div class="grid-container-desktop">
+          <ul class="usa-collection grid-row">
+            <div class="grid-row tablet:grid-gap-4">
+            {{/* Gets all communities pages */}}
+            {{ $communities := (where .Site.Pages "Section" "communities").ByTitle }}
+
+            {{/* Gather only the pages with a weight greater than or equal to 1 */}}
+            {{ $communities := (where $communities ".Params.weight" "ge" 1) }}
+
+            {{/* Gather only the pages with a dg-emphasis key */}}
+            {{ $communities := (where $communities ".Params.dg-highlight" "!=" true) }}
+
+            {{ range $i, $e := $communities }}
+            <div class="tablet:grid-col-6">
+              <li class="usa-collection__item">
+                <div class="usa-collection__body">
+                  <h3 class="usa-collection__heading">
```

Comment:
The **Other Communities of Practice** header is an h1, so this h3 breaks the order of headers. Let's use an h2 _or_ let's change the header for the Other section from h1 to h2.